### PR TITLE
prepare 3.4.1

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -390,7 +390,7 @@ manifest {
     description     = """An open-source analysis pipeline to detect germline or somatic variants from whole genome or targeted sequencing"""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
-    version         = '3.5dev'
+    version         = '3.14.1'
```
Comment (kenibrewer) [inline nextflow.config:393]: Minor fix needed here.
```suggestion
    version         = '3.4.1'
```

File: assets/multiqc_config.yml [hunk 2]
```diff
@@ -3,9 +3,9 @@ custom_logo_url: https://github.com/nf-core/sarek/
 custom_logo_title: "nf-core/sarek"
 
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/sarek/tree/dev" target="_blank">nf-core/sarek</a>
+  This report has been generated by the <a href="https://github.com/nf-core/sarek/releases/tag/3.14.1" target="_blank">nf-core/sarek</a>
```
Comment (kenibrewer) [inline assets/multiqc_config.yml:6]:
```suggestion
  This report has been generated by the <a href="https://github.com/nf-core/sarek/releases/tag/3.4.1" target="_blank">nf-core/sarek</a>
```

**Review by kenibrewer — APPROVED** [review]
Got some typos in the version number. Otherwise this looks good.

**Comment by maxulysse** [comment]
> Got some typos in the version number. Otherwise this looks good.

:facepalm:

**Review by kenibrewer — APPROVED** [review]

**Pull request merged**